locales/cy/translation.json

```diff
@@ -1454,6 +1454,43 @@
           "errorRadioMessage": "Dewiswch beth hoffech ei wneud"
         }
       }
+    },
+    "nonUkPassport": {
+      "title": "Do you have a passport with a biometric chip?",
+      "header": "Do you have a passport with a biometric chip?",
+      "content": {
+        "paragraph1": "You will need a passport with a biometric chip to prove your identity with GOV.UK One Login.",
+        "details": {
+          "summaryText": "How do I check if my passport has a biometric chip?",
+          "html": "<p>To check if your passport has a biometric chip, look for the rectangular biometric chip symbol on the front cover.</p><picture><source srcset=\"/public/images/passport-icon.svg\"/><img width=\"174\" height=\"121\" loading=\"lazy\" src=\"/public/images/passport-icon-1x.png\" srcset=\"/public/images/passport-icon-2x.png 2x\" alt=\"The front cover of a passport with the biometric chip symbol. The symbol is rectangular with a circle in the middle.\" /></picture>"
```

Comment:
We want to avoid as much html in the translations files as possible. I see elsewhere we call images straight into the template. Is this possible here?